src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -126,15 +225,17 @@ const ManageUsers: () => JSX.Element = () => {
                 <Tr>
                   <Th>{t('users.name')}</Th>
                   <Th>{t('email')}</Th>
-                  <Th>{t('users.isAdmin')}</Th>
+                  <Th>{t('users.adminStatus')}</Th>
```

Comment:
Change this text to be "add admin" instead (goes along with my next comment).

Reply:
Done.